### PR TITLE
[Admin cabinet] Add to the "Position" names division in Ukrainian and English

### LINKED ISSUE #5972: [Admin cabinet] Add to  the "Position" names division in Ukrainian and English
It is necessary to add English language to https://greencity-ubs.greencity.social/admin/ubs-employee/get-all-positions

Actual result:
({id: ..., name: "......"})

Expected result:
({id: ..., name: {en: "...", ua: "..."}})

Epic: [Admin https://github.com/ita-social-projects/GreenCity/issues/910]

### PR DESCRIPTION
**Pull request by ospodaryk**
[[Admin cabinet] Add to the "Position" names division in Ukrainian and English #5972]
(https://github.com/ita-social-projects/GreenCity/issues/5972)

Now Position have names division in Ukrainian and English

Added new Dto: PositionWithTranslateDto which returns map: :

Changed: New field in Position, PositionDto: "nameEN"

### CODE REVIEW

File: dao/src/main/java/greencity/entity/user/employee/Position.java [hunk 1]
```diff
@@ -22,6 +22,9 @@ public class Position {
     @Column(nullable = false, length = 30, unique = true)
     private String name;
 
+    @Column(nullable = false, length = 30, unique = true, name = "name_eng")
+    private String nameEN;
```
Comment (RomaMocherniuk) [inline dao/src/main/java/greencity/entity/user/employee/Position.java:26]: please change it to `nameEn`

File: service-api/src/main/java/greencity/dto/position/AddingPositionDto.java [hunk 2]
```diff
@@ -15,4 +15,6 @@ public class AddingPositionDto {
     @NotNull
     @Pattern(regexp = "[ЁёІіЇїҐґЄєА-Яа-яA-Za-z-'\\s.]{1,30}")
     private String name;
+    @Pattern(regexp = "[A-Za-z-'\\s.]{1,30}")
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/dto/position/AddingPositionDto.java:18]: please add tests for it

File: service-api/src/main/java/greencity/dto/position/PositionDto.java [hunk 3]
```diff
@@ -17,4 +18,6 @@ public class PositionDto {
     private Long id;
     @Pattern(regexp = "[ЁёІіЇїҐґЄєА-Яа-яA-Za-z-'\\s.]{1,30}")
     private String name;
+    @Pattern(regexp = "[A-Za-z-'\\s.]{1,30}")
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/dto/position/PositionDto.java:20]: please add tests

File: service-api/src/main/java/greencity/dto/position/PositionWithTranslateDto.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import javax.validation.constraints.Min;
+import java.util.Map;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@EqualsAndHashCode
+@ToString
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/dto/position/PositionWithTranslateDto.java:14]: please use `@Data`

**Review by Markiro1 — APPROVED** [review]

**Review by sergoliarnik — APPROVED** [review]

**Review by juseti — APPROVED** [review]

**Comment by juseti** [comment]
Please fix code smell

**Pull request merged**